### ISSUE TITLE
how can I use brave to trace the appication with RocketMQ?

### ISSUE THREAD

**Issue by ni-ze** [openzipkin/brave#1043]
this is my pom file: 
     
        <dependency>
            <groupId>org.apache.rocketmq</groupId>
            <artifactId>rocketmq-client</artifactId>
            <version>4.4.0</version>
        </dependency>
and I use like this:
```
   @Autowired
    private DefaultMQProducer defaultMQProducer;

   ...

   defaultMQProducer.send(message);
```

**Comment by adriancole** [comment]
unless rocketmq has a JMS driver, this would be a new instrumentation. We'd need a few others interested and a volunteer to write it

**Comment by adriancole** [comment]
https://github.com/apache/rocketmq/wiki/RIP-6-Message-Trace has some summary information about how RocketMQ thinks about tracing.

fields they record are listed here https://github.com/apache/rocketmq/blob/master/client/src/main/java/org/apache/rocketmq/client/trace/TraceBean.java

likely we minimally need topic messageKey, messageId properties to link to their trace data


cc also @literatureandyou in case you were looking for rocket here

**Comment by JoeKerouac** [comment]
@codefromthecrypt 
I plan to add rocketmq support in brave, is there anything I need to pay attention to?

**Comment by codefromthecrypt** [comment]
Is anyone still interested in this? If so, please 👍 this comment.

**Comment by CodePrometheus** [comment]
@codefromthecrypt Excuse me, it seems that RocketMQ has not been fully implemented from 2023 until now, I‘d like to build upon the existing work to complete its implementation.

**Comment by codefromthecrypt** [comment]
@CodePrometheus yes you can. you can start over or cherry-pick commits from the existing PR. Regardless, please review the existing PR notes carefully to reduce redudant work. Also, it may be easier to publish a docker image for rocketmq in zipkin's main repo if there are still no dual architecture images available from the canonical repo. Thanks for offering to progress this!
